doc/nrf/app_dev/ap_protect/index.rst

```diff
@@ -26,16 +26,16 @@ For detailed information, refer to the hardware documentation.
 .. list-table:: AP-Protect implementations
    :header-rows: 1
 
-   * - Implementation type
+   * - AP-Protect implementation type
      - Default factory state
      - How to enable
      - How to disable
-   * - AP-Protect controlled by hardware
+   * - Hardware
      - Disabled
      - Writing ``UICR.APPROTECT`` to ``Enabled`` and performing reset.
      - Issuing an ``ERASEALL`` command via CTRL-AP.
        This command erases the flash, UICR, and RAM, including ``UICR.APPROTECT``.
-   * - AP-Protect controlled by hardware and software
+   * - Hardware and software
      - Enabled
      - After it is disabled, a reset or a wake enables the access port protection again.
```

Comment:
```suggestion
     - When disabled, a reset or a wake enables the access port protection again.
```
This sentence feels like it was taken from a more running text that told how to disable it.

Should consideration be given to the second half of the disabling instructions?